prism-agent/service/server/src/main/scala/io/iohk/atala/pollux/credentialschema/VerificationPolicyEndpoints.scala

```diff
@@ -17,11 +18,23 @@ import java.util.UUID
 
 object VerificationPolicyEndpoints {
 
+  private val tagName = "Verification"
+  private val tagDescription = """
+    |<p>The `Verification` endpoints enable the management and querying of verification policies,
+    |which are applied to W3C Verifiable Credentials in JWT format.</p>
+    |<p>Users can retrieve and paginate existing policies or create new ones.
+    |These policies determine the verification criteria, allowing users to specify constraints such as `schemaId` and `trustedIssuers` in the current implementation.</p>
+    |<p>The constraints are defined using the `schemaId` and a sequence of `trustedIssuers`.
+    |This functionality ensures the system's integrity and adherence to specific verification requirements.</p>
+    |<p>Endpoints are secured by `apiKeyAuth` or `jwtAuth` authentication</p>""".stripMargin
```

Comment:
Something I notice locally is the swagger-ui refuse to render backtick as code block. It needs to be wrapped with html like `<code>apiKeyAuth</code>`. Last mile to "beautify" the apidoc will set a good example for the rest to follow :)

Reply:
you are right. let's collect the feedback and ask other people to review the intermediate version.
we also need to use <p> and <br> in a right way (already started discussion with @bvoiturier)